sql/conseil.sql

```diff
@@ -585,6 +590,7 @@ CREATE INDEX endorsing_rights_level_idx ON tezos.endorsing_rights USING btree (l
 
 CREATE INDEX endorsing_rights_delegate_idx ON tezos.endorsing_rights USING btree (delegate);
 
+CREATE INDEX ix_delegate_slot ON tezos.endorsing_rights USING btree (delegate);
```

Comment:
There is no actual slot in the definition of the index.

Reply:
Fixed.